cypress-io/cypress
# misc: add CTA footer to launchpad framework dropdown

- Closes #25799

### Additional details


We want to let users know that we support third-party integrations for CT frameworks. This PR adds a footer to the bottom of the framework options list in the launchpad that prompts the user to go to the docs to learn more about third-party support for frameworks.

### Steps to test


1. Open Cypress with this branch
2. Select a project that is not configured for component testing
3. Select component testing
4. Expand the framework dropdown and scroll to the bottom of the list
5. Verify that the footer appears at the bottom

The on link doesn't go anywhere yet - I'll add that once #25800 is in review so that I know where to link 

### How has the user experience changed?

<img width="1193" alt="Screenshot 2023-02-16 at 10 45 56 AM" src="https://user-images.githubusercontent.com/14275198/219446486-e329b887-d044-460e-b4fc-5d52d9ba0486.png">

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
